src/pages/Home.tsx

```diff
@@ -267,6 +267,9 @@ function TotalFeesCard(props: TotalFeesCardProps) {
           )
         })}
       </HStack>
+      <Box fontSize="14px" mt="0" color="brand.secondaryText">
+          since {timeSince(1624500217)}
```

Comment:
Can we add this timestamp to `config.ts`, maybe we can name it `eipActivationTime`, for the other networks, they can just be 0 for now.

Then in this function we can add the following before `useCurrency()`:

```
const network = useSetting<EthereumNetwork>(Setting.network)
```

And then just query it via:

```
since {timeSince(network.eipActivationTime)}
```